### PR TITLE
Fix/open graph - use gatsby head

### LINKED ISSUE #2475: Fix OpenGraph (`og:`) tags, head metadata for homepage + other pages
Currently, the homepage and other pages of AIID don't have the correct OpenGraph tags to enable previews, and has only "Open Twitter" metadata. This may be a problem with the React/AIID helmet code.

https://www.opengraph.xyz/url/https%3A%2F%2Fincidentdatabase.ai

```html
<!-- HTML Meta Tags -->
<title>Open Twitter</title>
<meta name="description" content="undefined">

<!-- Facebook Meta Tags -->
<meta property="og:url" content="https://incidentdatabase.ai">
<meta property="og:type" content="website">
<meta property="og:title" content="Open Twitter">
<meta property="og:description" content="undefined">
<meta property="og:image" content="">

<!-- Twitter Meta Tags -->
<meta name="twitter:card" content="summary_large_image">
<meta property="twitter:domain" content="incidentdatabase.ai">
<meta property="twitter:url" content="https://incidentdatabase.ai">
<meta name="twitter:title" content="Open Twitter">
<meta name="twitter:description" content="undefined">
<meta name="twitter:image" content="">

<!-- Meta Tags Generated via https://www.opengraph.xyz -->
```

Blog pages are incomplete titles:
image
https://incidentdatabase.ai/blog/incident-report-2023-may-june/
```html
<!-- HTML Meta Tags -->
<title>Open Twitter</title>
<meta name="description" content="undefined">

<!-- Facebook Meta Tags -->
<meta property="og:url" content="https://incidentdatabase.ai/blog/incident-report-2023-may-june/">
<meta property="og:type" content="website">
<meta property="og:title" content="Open Twitter">
<meta property="og:description" content="undefined">
<meta property="og:image" content="https://incidentdatabase.aihttps://images.prismic.io/aiidprod/200a219d-a4db-44a0-8a23-88329b9e5ec8_Group%20(1).png?ixlib=gatsbyFP&auto=compress%2Cformat&fit=max&w=1920&h=1080">

<!-- Twitter Meta Tags -->
<meta name="twitter:card" content="summary_large_image">
<meta property="twitter:domain" content="incidentdatabase.ai">
<meta property="twitter:url" content="https://incidentdatabase.ai/blog/incident-report-2023-may-june/">
<meta name="twitter:title" content="Open Twitter">
<meta name="twitter:description" content="undefined">
<meta name="twitter:image" content="https://incidentdatabase.aihttps://images.prismic.io/aiidprod/200a219d-a4db-44a0-8a23-88329b9e5ec8_Group%20(1).png?ixlib=gatsbyFP&auto=compress%2Cformat&fit=max&w=1920&h=1080">

<!-- Meta Tags Generated via https://www.opengraph.xyz -->
```

Other page seem fine, e.g. `cite` pages:
https://incidentdatabase.ai/cite/614/#r3445
https://www.opengraph.xyz/url/https%3A%2F%2Fincidentdatabase.ai%2Fcite%2F614%2F%23r3445

```html

  <!-- HTML Meta Tags -->
  <title>Incident 614: Google Bard Allegedly Generates False Allegations Against Consulting Firms Used in Research Presented in Australian Parliamentary Inquiry</title>
  <meta name="description" content="Australian academics reportedly used Google Bard AI to generate case studies for a parliamentary inquiry, leading to false allegations against major consultancy firms. The AI-generated misinformation prompted an apology from the academics, causing reputational harm for all parties involved and raising concerns about the reliability of AI tools in producing accurate and unbiased information.">

  <!-- Facebook Meta Tags -->
  <meta property="og:url" content="https://incidentdatabase.ai/cite/614/#r3445">
  <meta property="og:type" content="website">
  <meta property="og:title" content="Incident 614: Google Bard Allegedly Generates False Allegations Against Consulting Firms Used in Research Presented in Australian Parliamentary Inquiry">
  <meta property="og:description" content="Australian academics reportedly used Google Bard AI to generate case studies for a parliamentary inquiry, leading to false allegations against major consultancy firms. The AI-generated misinformation prompted an apology from the academics, causing reputational harm for all parties involved and raising concerns about the reliability of AI tools in producing accurate and unbiased information.">
  <meta property="og:image" content="https://res.cloudinary.com/pai/image/upload/v1/reports/i.guim.co.uk/img/media/f63d5f7c5919e31dca5f8c1761b183ad5175e3c4/1099_640_4130_2478/master/4130.jpg%3Fwidth%3D1200&height%3D630&quality%3D85&auto%3Dformat&fit%3Dcrop&overlay-align%3Dbottom%252Cleft&overlay-width%3D100p&overlay-base64%3DL2ltZy9zdGF0aWMvb3ZlcmxheXMvdGctZGVmYXVsdC5wbmc&enable%3Dupscale&s%3Da1b42dc840748f7777d19cc4ae1c43bd?_a=ATAABd+0">

  <!-- Twitter Meta Tags -->
  <meta name="twitter:card" content="summary_large_image">
  <meta property="twitter:domain" content="incidentdatabase.ai">
  <meta property="twitter:url" content="https://incidentdatabase.ai/cite/614/#r3445">
  <meta name="twitter:title" content="Incident 614: Google Bard Allegedly Generates False Allegations Against Consulting Firms Used in Research Presented in Australian Parliamentary Inquiry">
  <meta name="twitter:description" content="Australian academics reportedly used Google Bard AI to generate case studies for a parliamentary inquiry, leading to false allegations against major consultancy firms. The AI-generated misinformation prompted an apology from the academics, causing reputational harm for all parties involved and raising concerns about the reliability of AI tools in producing accurate and unbiased information.">
  <meta name="twitter:image" content="https://res.cloudinary.com/pai/image/upload/v1/reports/i.guim.co.uk/img/media/f63d5f7c5919e31dca5f8c1761b183ad5175e3c4/1099_640_4130_2478/master/4130.jpg%3Fwidth%3D1200&height%3D630&quality%3D85&auto%3Dformat&fit%3Dcrop&overlay-align%3Dbottom%252Cleft&overlay-width%3D100p&overlay-base64%3DL2ltZy9zdGF0aWMvb3ZlcmxheXMvdGctZGVmYXVsdC5wbmc&enable%3Dupscale&s%3Da1b42dc840748f7777d19cc4ae1c43bd?_a=ATAABd+0">

  <!-- Meta Tags Generated via https://www.opengraph.xyz -->
```

h/t @bweir for the issue!

### LINKED ISSUE #2655: Use prismic blog thumbnail image as metaImage

### PR DESCRIPTION
**Pull request by clari182**
- closes #2475 
- closes #2655

### CODE REVIEW

**Comment by clari182** [comment]
@cesarvarela please hold on review thanks!

File: site/gatsby-site/src/components/AiidHead.js [hunk 1]
```diff
@@ -53,9 +52,9 @@ const AiidHelmet = ({
         <link rel="shortcut icon" type="image/svg" href={config.siteMetadata.favicon} />
       )}
 
-      {children}
-    </Helmet>
+      {children && <>{children}</>}
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/AiidHead.js:55]: Why this change?

**Review by cesarvarela — CHANGES_REQUESTED** [review]
Needs to remove `react-helmet` from `package.json`

Considering that we have talked in the past about removing specific references to AIID #1957, this might be a good opportunity to use a more generic name.

File: site/gatsby-site/cypress/e2e/integration/submit.cy.js [hunk 2]
```diff
@@ -984,7 +984,7 @@ describe('The Submit form', () => {
     };
 
     for (const key in valuesStep1) {
-      cy.get(`[name="${key}"]`).type(valuesStep1[key]);
+      cy.get(`input[name="${key}"]`).type(valuesStep1[key]);
```
[the remark this answers is not in the document]
Comment (clari182) [reply site/gatsby-site/cypress/e2e/integration/submit.cy.js:987]: Yes, the open graph `title`. Now that it exists there are 2 elements with the `title` name

[the remark this answers is not in the document]
Comment (clari182) [reply site/gatsby-site/cypress/e2e/integration/submit.cy.js]: You're right, that was wrongly defined!

File: site/gatsby-site/src/components/AiidHead.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import Helmet from 'react-helmet';
 import config from '../../config';
 
-const AiidHelmet = ({
+const AiidHead = ({
   children,
   metaTitle = null,
-  metaDescription = null,
+  metaDescription = metaTitle,
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/AiidHead.js:7]: This is a weird default. All pages should have a title and a description for SEO purposes besides just correctness.  If one of these is missing, it is okay if the build crashes.

File: site/gatsby-site/src/components/AiidHead.js [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
   path,
   metaImage = null,
   metaType = 'website',
+  title = null,
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/AiidHead.js:11]: We should use either `title` or `metaTitle` but not both since they refer to the same thing.

**Comment by clari182** [comment]
@cesarvarela this is ready for review

**Review by cesarvarela — COMMENTED** [review]
Some open graph related test are failing, is it flake?

image

**Comment by clari182** [comment]
Fixed @cesarvarela

**Review by cesarvarela — APPROVED** [review]

**Comment by kepae** [comment]
Good to merge and test in staging @clari182 ?

**Comment by kepae** [comment]
Please resolve conflicts and let me know when I can merge this. Sorry if the other PRs forced this out of sync.

**Pull request merged**